recipe/meta.yaml

```diff
@@ -1,23 +1,22 @@
-{% set version = "0.2.3" %}
+{% set version = "0.2.6" %}
 
 package:
   name: pygeogrids
   version: {{ version }}
 
 source:
-  fn: pygeogrids-{{ version }}.tar.gz
   url: https://pypi.io/packages/source/p/pygeogrids/pygeogrids-{{ version }}.tar.gz
-  sha256: ad014bae48fc1666b8e56b8714fd0e00775effcd57b6159791d642f1522c7d6c
+  sha256: 935a30678749fced06d7578c4354cdf5acd75c5cf0223f33e9f81e8c0a188d98
 
 build:
   number: 0
-  script: python setup.py install --single-version-externally-managed --record record.txt
+  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
 
 requirements:
   build:
     - python
-    - setuptools
-    - pyscaffold
+    - pip
+    - pyscaffold >=2.5a0,<2.6a0
```

Comment:
To build this we need to satisfy that, but those versions constraints are too old and we cannot build them at the moment. Hopefully a newer version of `pygeogrids` will support latest `pyscaffold`, or better yet, drop `pyscaffold`.

Reply:
Thanks. The underlying problem is that theses are all python only dependencies which are dragged into conda.

How can I remove a package from conda-forge? I think we don't need `pygeogrids`, `pygeobase` etc. The only reason I started these packages was to in the end get `pytesmo` into conda forge since it has a compiled extension. But maintaining 4 or 5 other python only packages in conda forge does not seem to be worth the effort.

Reply:
You can just close this PR and list what you want to archive.